manager/integration/tests/test_upgrade.py

```diff
@@ -499,18 +521,24 @@ def test_upgrade(longhorn_install_method,
         write_volume_random_data(vol_revision_disabled)
     vol_rebuild_data_after_sys_upgrade = \
         write_volume_random_data(vol_rebuild)
+    strict_local_vol_data_after_sys_upgrade = \
+        write_volume_random_data(strict_local_vol)
 
     # Check data written to all volumes
     for sspod_info in statefulset_pod_info:
         resp = read_volume_data(core_api, sspod_info['pod_name'])
-        assert resp == sspod_info['data']
+        assert resp == sspod_info['data'], \
+            f"assert data can be written to {sspod_info['pod_name']} \
+              after upgrade"
 
     check_volume_data(vol_revision_enabled,
                       vol_revision_enabled_data_after_sys_upgrade)
     check_volume_data(vol_revision_disabled,
                       vol_revision_disabled_data_after_sys_upgrade)
     check_volume_data(vol_rebuild,
                       vol_rebuild_data_after_sys_upgrade)
+    check_volume_data(strict_local_vol,
+                      strict_local_vol_data_after_sys_upgrade)
 
     # Detach the vol_revision_enabled & vol_revision_disabled,
```

Comment:
Can we add comment about local-strict volume here?

Reply:
Added. Thank you!

Reply:
I meant, adding the comment about the detach, so that later it will be easier to understand that the strict-local volume is also getting detached. Going ahead a merging this, this should not clock the merge. 👍 
`Detach the vol_revision_enabled, strict_local_vol vol_revision_disabled`.